macros/dbt_utils/sql/deduplicate.sql

```diff
@@ -0,0 +1,20 @@
+{%- macro trino__deduplicate(relation, partition_by, order_by) -%}
+    {#
+    -- This is a temporary solution until https://github.com/trinodb/trino/issues/14455 is resolved
+    -- It leaks the rn column compared to the NATURAL JOIN solution from dbt
```

Comment:
What it means `rn`?

Reply:
So the NATURAL JOIN implementation (which is not supported) would not leak the row number column as output of the deduplicate query.